homework.py

```diff
@@ -0,0 +1,41 @@
+import random
+from time import sleep
+from datetime import datetime
+def retry(Exceptions, tries,delay,backoff):
+
+    def decorator(func):
+
+        def wrapper(*args, **kwargs):
+            nonlocal func,delay
+            for i in range(1,tries+1):
+                print(f"{datetime.now()}::Retrying {func.__name__} {i} time!!")
+                try:
+                    func(*args, **kwargs)
```

Comment:
You need to get the result of the function that was decorated, in this case a `return` statement would be a good helper